src/charm.py

```diff
@@ -17,37 +18,60 @@
 from charms.prometheus_k8s.v0.prometheus_scrape import MetricsEndpointProvider
 from charms.velero_libs.v0.velero_backup_config import VeleroBackupProvider, VeleroBackupSpec
 from lightkube import ApiError, codecs
-from lightkube.generic_resource import load_in_cluster_generic_resources
+from lightkube.generic_resource import create_global_resource, load_in_cluster_generic_resources
 from lightkube.models.core_v1 import ServicePort
 from lightkube.resources.core_v1 import Namespace
+from lightkube.types import PatchType
 from ops import main
 from ops.charm import ActionEvent, CharmBase
 from ops.framework import StoredState
 from ops.model import ActiveStatus, BlockedStatus, Container, MaintenanceStatus, WaitingStatus
 from ops.pebble import ChangeError, Layer
+from pydantic import ValidationError
 from serialized_data_interface import NoCompatibleVersions, NoVersionsListed, get_interfaces
 
 from constants import (
     K8S_RESOURCE_FILES,
     K8S_USER_WORKLOAD_EXCLUDE_RESOURCECS,
     NAMESPACE_LABELS_FILE,
 )
+from models import CharmConfig
+
+ProfileLightkube = create_global_resource(
```

Comment:
suggestion (blocking): perhaps just an oversight, shall we clean it if no longer used?